### PR TITLE
Create MariaDB droplet-1click app

### PR DESCRIPTION
**Pull request by alexeigutium**
This PR adds required packer files for the new app - MariaDB

### CODE REVIEW

**Review by scott — APPROVED** [review]
LGTM!

File: mariadb-22-04/scripts/010-mariadb.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Scripts in this directory are run during the build process.
```
Comment (v-aisac) [inline mariadb-22-04/scripts/010-mariadb.sh:3]: ```Scripts in this directory are run```
->
```Scripts in this directory run```

File: mariadb-22-04/scripts/010-mariadb.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# remove the scripts from your build system after they have run
+# if you use the build_image task.
+#
+
+systemctl start mariadb.service
+# Add first-login task
+cat >> /root/.bashrc <<EOM
+/opt/digitalocean_mariadb/setup_mariadb.sh
+EOM
+
```
Comment (v-aisac) [inline mariadb-22-04/scripts/010-mariadb.sh:15]: Extra newline?

File: mariadb-22-04/template.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (v-aisac) [inline mariadb-22-04/template.json:1]: Extra newline too?

**Review by v-aisac — APPROVED** [review]
Looks good, just few nitpicks

**Pull request merged**